### ISSUE TITLE
Request for MELD Dataset .pkl Files and Hyperparameter Settings for Reproducing EIMOCAP Results

### ISSUE THREAD

**Issue by fiyud** [Fsoft-AIC/ConxGNN#1]
I am currently working on reproducing the results from the paper regarding emotion recognition using the EIMOCAP dataset. While trying to replicate the reported results, I was only able to achieve around 67% accuracy, compared to the 68.52% stated in the paper.

Can you help me by provide the .pkl files for the MELD dataset used in the experiments.

And the exact hyperparameter settings used for training on the EIMOCAP dataset, including any learning rate, batch size, optimizer details, and training epochs.

**Comment by thp1228** [comment]
> 我目前正在利用EIMOCAP数据集重现论文中关于情感识别的结果。在尝试复现报告结果时，我只获得了大约67%的准确率，而论文中称的68.52%。
> 
> 你能帮我提供实验中使用的MELD数据集的.pkl文件吗？
> 
> 以及用于EIMOCAP数据集训练的具体超参数设置，包括任何学习率、批次大小、优化器细节和训练历程。

Hello! Do you have the dataset yet?

**Comment by fiyud** [comment]
No, I haven't
